### PR TITLE
Log warning if wrong PLUGIN_SET format supplied

### PR DESCRIPTION
**Pull request by ArnoStiefvater**

### CODE REVIEW

File: ospd_openvas/daemon.py [hunk 1]
```diff
@@ -377,6 +377,12 @@ def feed_is_outdated(self, current_feed: str) -> Optional[bool]:
             return None
 
         current_feed = safe_int(current_feed)
+        if current_feed is None:
+            logger.warning(
+                "Wrong PLUGIN_SET format in plugins feed file %s. Format has to"
```
Comment (cfi-gb) [inline ospd_openvas/daemon.py:382]: Could it be possible that this happens if the file isn't found at all?

Comment (ArnoStiefvater) [reply ospd_openvas/daemon.py:382]: There is already a debug message a few lines above for this case. `logger.debug('Plugins feed file %s not found.', feed_info_file)`.
Should I change the debug level to warning?

Comment (cfi-gb) [reply ospd_openvas/daemon.py:382]: If ospd-openvas can't work without an existing / accessible `plugin_feed_info.inc` then i would raise this debug to warning.

Comment (ArnoStiefvater) [reply ospd_openvas/daemon.py:382]: I raised the level to warning now.

**Pull request merged**